internal/runner/result.go

```diff
@@ -0,0 +1,14 @@
+package runner
+
+type TestStatus string
+
+const (
+	TestStatusPassed TestStatus = "passed"
+	TestStatusFailed TestStatus = "failed"
+	TestStatusError  TestStatus = "error"
+)
+
+type TestResult struct {
```

Comment:
nit: My initial impression about this struct is that it holds the result of a single test case. Perhaps we could name it differently to reflect it holds the result of executing the test command.

Reply:
Good point. Since this inside the `runner` package, and it refers to the status of the runner (e.g. Rspec, Jest) Maybe we can just call it `Result`? Let me know what you think, or if you have a better suggestion.

```
result, _ := testRunner.Run(...)
if result.Status == runner.StatusFailed {
  ...
}
```

Reply:
`RunExecutionResult`? Since it's the result of calling `Run`?